ledger/src/blockstore.rs

```diff
@@ -751,6 +913,61 @@ impl Blockstore {
         );
     }
 
+    /// Collects and reports [`BlockstoreRocksDbColumnFamilyMetrics`] for the
+    /// all the column families.
+    pub fn submit_rocksdb_cf_metrics_for_all_cfs(&self) {
+    }
+
+    /// Collects and reports [`BlockstoreRocksDbColumnFamilyMetrics`] for the
+    /// given column family.
+    fn submit_rocksdb_cf_metrics<C: 'static + Column + ColumnName>(
+        &self,
+        metric_name: &'static str,
+    ) {
+        let cf = self.db.column::<C>();
+        let cf_rocksdb_metrics = BlockstoreRocksDbColumnFamilyMetrics {
+            total_sst_files_size: cf
+                .get_int_property(RocksProperties::TOTAL_SST_FILES_SIZE)
+                .unwrap_or(0),
```

Comment:
For some of these properties (probably not this one specifically lol), a value of `0` might be a perfectly valid value; thoughts on doing `.unwrap_or(-1)` so we can more easily identify property read errors?

Reply:
Good point, although in the report step, we are explicitly submitting an `i64`
```
            (
                "total_sst_files_size",
                self.total_sst_files_size as i64,
                i64
            ),
```
If we're going to cast anyways, we could update the struct to have `i64`'s and move the casts to populating the struct instead of reporting the struct

Reply:
Yep, we can do that.  I will at least convert those properties which have 0 chance to overflow i64.

Reply:
Max value of an i64 is 9223372036854775807 or 9.22 x 10^18; I don't think any of these fields would be overflowing that right ?